assets/js/game.js

```diff
@@ -0,0 +1,20 @@
+function Game() {
```

Comment:
When you create a new game, maybe start with the starting character? So `function Game(hero) { this.heroes = [hero]; //more`